threenums.c

```diff
@@ -0,0 +1,31 @@
+/*Ashank Kumar*/
```

Comment:
Perfect job, but, again, add description of your program at the top or comments throughout!